common/gossmap.h

```diff
@@ -44,7 +44,17 @@ struct gossmap_chan {
 /* If num_channel_updates_rejected is not NULL, indicates how many channels we
  * marked inactive because their values were too high to be represented. */
 #define gossmap_load(ctx, filename, logcb, cbarg)			\
-	gossmap_load_((ctx), (filename),				\
+	gossmap_load_((ctx), (filename), (0),				\
+		      typesafe_cb_postargs(void, void *, (logcb), (cbarg), \
+					   enum log_level,		\
+					   const char *fmt,		\
+					   ...),			\
+		      (cbarg))
+
+/* If we're the author of the gossmap, it should have no redundant records, corruption, etc.
+ * So this fails if that's not the case. */
```

Comment:
Making Jon Postel proud! This is cool solution to what we were missing when gossmap took over the gossip_store.